src/libs/actions/IOU.js

```diff
@@ -44,6 +44,13 @@ Onyx.connect({
     },
 });
 
+let allRecentlyUsedCategories = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY_RECENTLY_USED_CATEGORIES,
```

Comment:
Just to clarify, we are getting all all categories for all policies here?

Reply:
Yes, similar to what's being done above for reports and transactions.